.golangci.yml

```diff
@@ -38,10 +38,21 @@ linters:
 issues:
   max-same-issues: 0
   max-issues-per-linter: 0
+  # We are disabling default golangci exclusions because we want to help reviewers to focus on reviewing the most relevant
+  # changes in PRs and avoid nitpicking.
+  exclude-use-default: false
   # List of regexps of issue texts to exclude, empty list by default.
   exclude:
     - Using the variable on range scope `(tc)|(rt)|(tt)|(test)|(testcase)|(testCase)` in function literal
     - "G108: Profiling endpoint is automatically exposed on /debug/pprof"
+    - Error return value of .((os\.)?std(out|err)\..*|.*Close|.*Flush|os\.Remove(All)?|.*print(f|ln)?|os\.(Un)?Setenv). is not checked
```

Comment:
I wonder if we want to get rid of a few of them. not the ones about `fmt.Fprintf`, but the e.g. the `File.Close()`

Reply:
As mentioned above, file.Close() in a defer is actually useful. While ignoring the error can be odd, if a file cannot be closed (because it's a syscall) an error is usually thrown way earlier.

Reply:
Okay, missed your comment above. Then it's fine for me. 

Just had a few of those when close failed when writing to S3/GCS and it's pretty hard to debug. But we also have "manual' reviews, so we should be good